en/docs/reference/customize-product/extending-api-manager/extending-workflows/configuring-http-redirection-for-workflows.md

```diff
@@ -33,8 +33,7 @@ To customize the default workflow extension, you override the **`execute()`** an
        public WorkflowResponse execute(WorkflowDTO workflowDTO) throws WorkflowException {
            super.execute(workflowDTO);
     ```
-4.  In the **`execute()`** method define the `CallbackUrl`, `RedirectUrl`, `workflowRefId`, and `RedirectConfirmationMsg`. The CallbackUrl should ideally be an open endpoint accepting workflowRefId as a parameter. Once the endpoint is invoked, it should in-turn invoke the [Admin REST APIs callback method]({{base_path}}/reference/product-apis/admin-apis/admin-v2/admin-v2/#tag/Workflows-(Individual)/paths/~1workflows~1update-workflow-status/post). Note that the Admin REST API resources require authentication before invocation.
-    ``` java
+4.  In the **`execute()`** method define the `CallbackUrl`, `RedirectUrl`, `workflowRefId`, and `RedirectConfirmationMsg`. The CallbackUrl should ideally be an open endpoint accepting workflowRefId as a parameter. Once the endpoint is invoked, it should in-turn invoke the [Admin REST APIs callback method]({{base_path}}/reference/product-apis/admin-apis/admin-v3/admin-v3/#tag/Workflows-(Individual)/paths/~1workflows~1update-workflow-status/post)
```

Comment:
As the product version is APIM 4.0.0, the correct link would be `reference/product-apis/admin-apis/admin-v2/admin-v2/#tag/Workflows/paths/~1workflows~1update-workflow-status/post`